### PR TITLE
Add Tests For The Integrity Check

### PR DESCRIPTION
**Pull request by irisk29**
I have added tests for the integrity check with the validate all option -
1. good scenario - simply activate the integrity check
2. negative scenario - delete one block and run the integrity check

Moreover, I have modified the integrity check to return a value upon success/failure.

### CODE REVIEW

File: tests/apollo/test_skvbc_s3_integrity_check.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        client = Minio("127.0.0.1:9000", access_key="concordbft", secret_key="concordbft", secure=False)
+        buckets = client.list_buckets()
+        bucketName = buckets[0].name
+        try:
+            client.remove_object(bucketName, 'concord/blocks/1')
+        except ResponseError as err:
+            assert False
+
+    async def _wait_for_st(self, bft_network, ro_replica_id, seqnum_threshold=150):
+        # TODO replace the below function with the library function:
```
Comment (toly-kournik) [inline tests/apollo/test_skvbc_s3_integrity_check.py:185]: why not TODO now? :-)

Comment (irisk29) [reply tests/apollo/test_skvbc_s3_integrity_check.py]: removed this function and used `await bft_network.wait_for_state_transfer_to_start()` instead

File: tests/apollo/test_skvbc_s3_integrity_check.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+from util.test_base import ApolloTest
+from util import skvbc as kvbc
+from util import eliot_logging as log
+from util.object_store import ObjectStore, start_replica_cmd_prefix
+from util.bft import KEY_FILE_PREFIX, with_trio, with_bft_network
+
+def start_replica_cmd(builddir, replica_id, config):
+    """
+    There are two test s3 config files. The one used here hasn't got s3-prefix parameter
+    set which means that on each RO replica start the prefix is some unique time based
```
Comment (toly-kournik) [inline tests/apollo/test_skvbc_s3_integrity_check.py:28]: This was fixed recently: if not provided, prefix will be an empty string

Comment (irisk29) [reply tests/apollo/test_skvbc_s3_integrity_check.py]: fixed.

**Review by toly-kournik — APPROVED** [review]
LGTM!
thanks @irisk29

**Pull request merged**